lib/resolver.js

```diff
@@ -149,7 +179,14 @@ SchemaResolver.prototype.resolveRef = function (ref) {
     }
 
     if (!dest || typeof dest !== 'object') {
-        throw err;
+        if (missing$Ref) {
+            this.missing.push(ref);
+            dest = {
+                additionalProperties: true
```

Comment:
I'm not sure about this part, nested $ref are default additionalProperties=true (so not require that) even if parent isn't. Is that correct?